infrastructure/resource_group_level/main.bicep

```diff
@@ -40,6 +40,10 @@ param vnetAddressPrefixes array = [
 ]
 
 param privateEndpointsCidr string = '172.16.20.0/24'
+
+param peeringRemoteVnetSubscriptionId string = '08887298-3821-49f0-8303-f88859c12b9b'
```

Comment:
this one needs to be an environment variable

Reply:
Yes - which is why I've made it a `param`, though I need to put in a default for my local deploys and for use until we have the GHA config & secrets set up.
My understanding is that the subscriptionId isn't considered secret.